### PR TITLE
RNA-FA/RNA_DNA updated, Extended version included

### PR DESCRIPTION
**Pull request by Arslan-Siraj**
## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out

- The details of the action statuses at the end of the PR or the "Checks" tab.
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. Use the "Show filters" capability on the top right to search for your PR number.
  If you click in the column that lists the failed tests you will get detailed error messages.

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

File: src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h [hunk 1]
```diff
@@ -858,48 +898,60 @@ namespace OpenMS
...
+    "G:C6H5N5O;G'+FA",
     "G:C11H14N5O8P;G+FA",
+    "G:C11H12N5O7P;G+FA-H2O",
+    "G:C11H11N5O4;G+FA-H3PO4",
     "G:C11H13N5O5;G+FA-HPO3",
+    "G:C7H5N5O;G'+2FA",
     "G:C12H14N5O8P;G+2FA",
     "G:C12H13N5O5;G+2FA-HPO3",
+    "G:C12H11N5O4;G+2FA-H3PO4",
+    "G:C12H13N5O5;G+2FA-HPO3",
```
Comment (timosachsenberg) [inline src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h:923]: same as in line 921?

File: src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h [hunk 2]
```diff
@@ -252,7 +252,25 @@ namespace OpenMS
     "A:+C4H6O2-NH3-H3PO4"
   };  
     
-  static constexpr std::array<const char*, 24> modifications_RNA_FA
+  static constexpr std::array<const char*, 12> modifications_RNA_FA
+  {
+    "G:C",
```
Comment (timosachsenberg) [inline src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h:257]: probably works but usually I add the plus sign
e.g. "G:+C"

File: src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h (hunk 1 above)
Comment (Arslan-Siraj) [reply src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h:923]: Ohh.. should be one of this "G:C12H13N5O6P;G+2FA-H2O",

File: src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h (hunk 2 above)
Comment (Arslan-Siraj) [reply src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h:257]: ok i will update

File: src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h (hunk 1 above)
Comment (timosachsenberg) [reply src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h:923]: k maybe double check it once more. I know it is super easy to make small mistakes here :)

**Review by timosachsenberg — APPROVED** [review]

**Pull request merged**